dls-controls/malcolmjs
# Feature/value archiving

## Description

Added archiving of attributes to malcom state in redux. Also added plot view for selected widget types.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- npm start
- navigate to localhost:3000/PANDA:INENC1/val
- view plot of archived data
- navigate to localhost:3000/PANDA:INENC1/a
- view plot of archived bool data (shows as 0/1 instead of true/false)

## Agile board tracking

connect to #263 
closes #263
